docs/glossary/README.md

```diff
@@ -0,0 +1,287 @@
+
+### Project type
+
+The project type in OpenProject is defined as an attribute that is assigned to a project. The project type allows the grouping of projects based on defined types. Project types are used in the timeline configuration.
+
+### Public project
+
+In OpenProject, projects can be private or public. Public means that the project is visible to any user regardless of project membership. The visibility of a project can be changed in the project settings. [Read how to set a project to public in OpenProject](https://www.openproject.org/docs/user-guide/projects/#set-a-project-to-public)
+
+## Repository
+
+A repository is defined as a document management system that allows users to manage files and folders via different version control system (SVN or Git).
+
+## Roadmap
+
+In OpenProject, a roadmap is defined as an overview page displaying the versions sorted alphabetically and the work packages assigned to them. The roadmap is displayed in the project navigation when the work package module is activated and a version has been created (project settings).
+
+## Role
+
+In OpenProject, a role is defined as a set of permissions defined by a unique name. Project members are assigned to a project by specifying a user’s, group’s or placeholder user’s name and the role(s) they should assume in the project.
+
+## Single user
+
+## Story points
+
+## Team planner
+
+## Time tracking
+
+Time tracking in OpenProject is defined as a module which allows users to log time on work packages. Once the time tracking module is activated, time can be logged via the action menu of a work package.
+
+## User
+
+In OpenProject, a user is defined as a person who uses OpenProject, described by an identifier. New users can be created in the admin settings. Users can become project members by assigning them a role and adding them via the project settings.
+
+## Widget
+
+## Wiki
+
+In OpenProject, a wiki is defined as a module that allows to use textile-based wiki pages. In order to use the wiki module, it has to be activated in the project settings.
+
+## Workflow
+
+A workflow in OpenProject is defined as the allowed transitions between status for a role and a type, i.e. which status changes can a certain role implement depending on the work package type. Workflows can be defined in the admin settings. For example, you might only developers to be able to set the status "developed". [Read more about work package workflows in OpenProject](https://www.openproject.org/docs/system-admin-guide/manage-work-packages/work-package-workflows/#manage-work-package-workflows)
+
+![glossary-openproject-work package](glossary-openproject-work package.png)
```

Comment:
⚠️ Please don't use spaces in image filenames.

⚠️ The image for work package is linked in section "Workflow"

Please provide a human readable image description. (e.g. this is read aloud by screenreaders)
e.g.
`![A work package in OpenProject](glossary-openproject-work-package.png)`

Please make sure that the file size of the image is in our [recommended limits](https://www.openproject.org/docs/development/contribution-documentation/documentation-style-guide/#save-images). e.g. this file is 460KB